README.md

```diff
@@ -68,6 +66,7 @@ If you use this program for your work, please cite
 
   - [1803.02846](https://arxiv.org/abs/1803.02846)
   - [2007.01725](https://arxiv.org/abs/2007.01725)
+  - [24XX.XXXXX](https://arxiv.org/abs/24XX.XXXXX)
```

Comment:
Do we want this here with placeholders or should we just make a new PR after the paper is released?

Reply:
Let's take it out and we push it with the manual.